.github/workflows/ci.yml

```diff
@@ -15,12 +15,14 @@ jobs:
       CI: true
     steps:
       - uses: actions/checkout@v2
-      - name: Use Node.js 13.x
+      - name: Use Node.js 12.x
```

Comment:
why the downgrade?

Reply:
This was from a separate PR that got merged into this one. There was a linting issue with the 13.x version but it didn't always happen. I'll eventually bump it up since I haven't seen that issue again.